core/sql/sqlcomp/PrivMgrMD.h

```diff
@@ -187,6 +202,25 @@ class ObjectUsage
   PrivMgrDesc originalPrivs;
   PrivMgrDesc updatedPrivs;
 
+  void copyColumnReferences(const std::vector<ColumnReference *> *refsToCopy)
+  {
+    if (refsToCopy == NULL)
```

Comment:
Might there be an old value in columnReferences? Do we need to do an NABASICDELETE (or a delete) on it before assigning a new object pointer to it?